From: Ben Hutchings <ben@decadent.org.uk>
Date: Fri, 5 Mar 2010 23:28:48 +0000
Subject: [PATCH] Correct attribution and licence of Sierra Wireless CIS files

These files have nothing to do with the pcmcia-cs project; they are
distributed by Sierra Wireless under GPLv3.

--- a/firmware/WHENCE
+++ b/firmware/WHENCE
@@ -634,13 +634,22 @@ File: cis/MT5634ZLX.cis
       cis/RS-COM-2P.cis
       cis/COMpad2.cis
       cis/COMpad4.cis
-      cis/SW_555_SER.cis
+
+Licence: GPL
+
+Originally developed by the pcmcia-cs project
+
+--------------------------------------------------------------------------
+
+Driver: SERIAL_8520_CS - Serial PCMCIA adapter
+
+File: cis/SW_555_SER.cis
       cis/SW_7xx_SER.cis
       cis/SW_8xx_SER.cis
 
-Licence: GPL
+Licence: GPLv3
 
-Originally developed by the pcmcia-cs project
+Copyright Sierra Wireless
 
 --------------------------------------------------------------------------
 
